packages/@lwc/synthetic-shadow/src/faux-shadow/node.ts

```diff
@@ -420,6 +420,15 @@ defineProperties(Node.prototype, {
     },
     contains: {
         value(this: Node, otherNode: Node): boolean {
+            if (DISABLE_NODE_PATCH) {
```

Comment:
fyi @caridy, @ekashida and @ravijayaramappa had to add `contains` and `cloneNode` behind the flag.